tailnet/configmaps.go

```diff
@@ -314,8 +349,17 @@ func (c *configMaps) reconfig(nm *netmap.NetworkMap) {
 		return
 	}
 
-	rc := &router.Config{LocalAddrs: nm.Addresses}
-	err = c.engine.Reconfig(cfg, rc, &dns.Config{}, &tailcfg.Debug{})
+	rc := &router.Config{
+		LocalAddrs: nm.Addresses,
+		Routes:     []netip.Prefix{CoderServicePrefixCIDR},
+	}
+	err = c.engine.Reconfig(cfg, rc, &dns.Config{
+		Routes: map[dnsname.FQDN][]*dnstype.Resolver{
+			CoderDNSSuffix: nil,
+		},
+		Hosts:    hosts,
```

Comment:
When the Resolver is `nil`, the route gets resolved using the `Hosts` field. The original issue mentioned 
> If enabled the Tailscale DNS service should respond on [fd60:627a:a42b::53]:53.
How do we know if it's running and whether or not we should supply that IP as the resolver here? Why can't we just always use this hosts field?